core/templates/pages/exploration-player-page/learner-experience/tutor-card.directive.ts

```diff
@@ -78,6 +78,7 @@ angular.module('oppia').directive('tutorCard', [
         onDismiss: '&',
         getDisplayedCard: '&displayedCard',
         startCardChangeAnimation: '=',
+        showHeaderIcon: '='
```

Comment:
'=' is generally used for variables and & for functions if i am not mistaken. And showHeaderIcon is just a boolean not a function that might change, so '=' better suits our need. What do you think? @seanlip

Reply:
That's not correct. '=' is used for 2-way binding and we use '&' for "readonly" binding (it also has the effect of converting a variable into a getter function -- see other examples in the codebase). Use '&' wherever you can )i.e. wherever you don't need 2-way binding) so that it's more performant.

Reply:
Ohh, thanks for explaning. Fixed this.